EnglishFilter/sections/general_extensions.txt

```diff
@@ -234,6 +234,9 @@ flashx.net#%#//scriptlet("abort-current-inline-script", "decodeURIComponent", "S
 cambay.tv#%#//scriptlet("set-constant", "flashvars.popunder_url", "")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/69639
 fappee.com#%#//scriptlet('abort-current-inline-script', 'doMyStuff')
+! https://github.com/AdguardTeam/AdguardFilters/issues/70037
+iceporn.com#%#("abort-on-property-write", "mobileAdvPop")
```

Comment:
It looks like that `//scriptlet` part is missed - `iceporn.com#%#//scriptlet("abort-on-property-write", "mobileAdvPop")`